doc/content/reference/id-eui-constraints/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "ID and EUI Constraints"
+description: ""
+---
+
+This reference explains how {{% tts %}} deals with IDs and EUIs, including regular expression requirements, uniqueness, and how deletion is handled.
+
+<!--more-->
+
+## Requirements of an ID or EUI
+
+An ID or EUI in {{% tts %}} must:
+
+- Have a length of 3-36 characters
+- Consist of lowercase letters, numbers, and non-consecutive dashes
```

Comment:
You cannot start with a dash either, but I don't know how we can spell this out in plain English. Maybe a 3rd rule that says 'must not start with a dash' ?